examples/plot_compare_reduction.py

```diff
@@ -42,14 +43,16 @@
         'classify__C': C_OPTIONS
     },
     {
-        'reduce_dim': [SelectKBest(chi2)],
+        'reduce_dim': [SelectKBest(chi2), SelectKBest(info_gain),
```

Comment:
I think repeated runs of digits might be too slow for our CI to run the plot. Some of that can be left to the user's imagination.